### PR TITLE
test: Add BDD feature file to test e2e transaction flow

### LINKED ISSUE #96: Add BDD feature file to test e2e transaction flow
Currently, BDD feature file just tests chaincode installation. This feature file need to be updated  to test transaction e2e flow.

### PR DESCRIPTION
**Pull request by rolsonquadras**
- Add e2e transaction flow feature in BDD file
- Update docker compose to enable CouchDB for state data
- Fix example chaincode 'del' function

Closes #96

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: test/bddtests/features/example.feature [hunk 1]
```diff
@@ -11,6 +11,41 @@ Feature:
...
+    And we wait 2 seconds
+    And client queries chaincode "examplecc" with args "get,k1" on a single peer in the "peerorg1" org on the "mychannel" channel
+    Then response from "examplecc" to client equal value ""
+
+
+    # Test private data collections transactions
+    When client invokes chaincode "examplecc" with args "putprivate,collection3,pvtKey,pvtVal" on the "mychannel" channel
+    And we wait 2 seconds
+    And client queries chaincode "examplecc" with args "getprivate,collection3,pvtKey" on a single peer in the "peerorg1" org on the "mychannel" channel
+    Then response from "ol_examplecc" to client equal value "pvtVal"
```
Comment (bstasyszyn) [inline test/bddtests/features/example.feature]: examplecc - not ol_examplecc

File: test/bddtests/features/example.feature [hunk 2]
```diff
@@ -11,6 +11,41 @@ Feature:
...
+
+    When client invokes chaincode "examplecc" with args "delprivate,collection3,pvtKey" on the "mychannel" channel
+    And we wait 2 seconds
+    And client queries chaincode "examplecc" with args "getprivate,collection3,pvtKey" on a single peer in the "peerorg1" org on the "mychannel" channel
+    Then response from "ol_examplecc" to client equal value ""
```
Comment (bstasyszyn) [inline test/bddtests/features/example.feature]: examplecc

**Comment by rolsonquadras** [comment]
> I think the name of the feature should be changed to e2e if it's going to be part of the tests.

@bstasyszyn Agreed. Renamed files/case from "example" to "e2e".

**Comment by rolsonquadras** [comment]
verify-bddtest

**Comment by rolsonquadras** [comment]
verify-bddtest

**Pull request merged**